integrations/terraform/tfschema/token/custom_types.go

```diff
@@ -0,0 +1,62 @@
+package token
+
+import (
+	"context"
+	"fmt"
+	apitypes "github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/teleport/integrations/terraform/tfschema"
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+// GenSchemaLabels returns Terraform schema for Labels type
+func GenSchemaLabels(ctx context.Context) tfsdk.Attribute {
+	return tfschema.GenSchemaLabels(ctx)
+}
+
+// GenSchemaBoolOptionsFixed returns Terraform schema for BoolOption type
+func GenSchemaBoolOptionFixed(_ context.Context) tfsdk.Attribute {
+	return tfsdk.Attribute{
+		Optional: true,
+		Type:     types.BoolType,
+	}
+}
+
+func CopyFromBoolOptionFixed(diags diag.Diagnostics, tf attr.Value, o **apitypes.BoolOption) {
+	v, ok := tf.(types.Bool)
+	if !ok {
+		diags.AddError("Error reading from Terraform object", fmt.Sprintf("Can not convert %T to types.Bool", tf))
+		return
+	}
+	if !v.Null && !v.Unknown {
+		value := apitypes.BoolOption{Value: v.Value}
+		*o = &value
+		return
+	}
+}
+
+func CopyToBoolOptionFixed(diags diag.Diagnostics, o *apitypes.BoolOption, t attr.Type, v attr.Value) attr.Value {
+	value, ok := v.(types.Bool)
+	if !ok {
+		value = types.Bool{}
+	}
+
+	if o == nil {
+		value.Null = true
+		return value
+	}
+
```

Comment:
I think we may need to explicitly set `value.Null` to False here ? My understanding is that if there's existing state/config for `v`, this will be pushed in and could already be set to Null.

Reply:
done